src/sentry/api/endpoints/organization_events_spans_histogram.py

```diff
@@ -0,0 +1,70 @@
+import sentry_sdk
```

Comment:
could you add your team to this file in `CODEOWNERS`? thanks!

Reply:
oh sorry, I didn't see the comment at the top of the file. Done 🙏